test/unittests.jl

```diff
@@ -2087,8 +2211,24 @@ end
 if :MySV2 ∉ UNIT_SYMBOLS
     @eval @register_unit MySV2 us"km/h"
 end
+if :psi ∉ UNIT_SYMBOLS
+    @eval @register_unit psi 6.89476us"kPa"
+end
+
+if :psig ∉ AFFINE_UNIT_SYMBOLS
+    @eval @register_affine_unit psig AffineDimensions(offset=u"Constants.atm", basedim=u"psi")
+else
+    skipped_register_unit = true
```

Comment:
I don't think this does anything. At this point in the code it should already be true. Please delete